app/api/tax.py

```diff
@@ -28,92 +29,94 @@ def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
         if not has_access('is_coorganizer', event_id=data['event']):
             raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
+        if get_count(db.session.query(Event).filter_by(id=int(data['event']), is_tax_enabled=False)) > 0:
```

Comment:
Just curious, why did you use `db.session.query()` here and `self.session.query()` in `before_create_object()` [line 43]?

Reply:
No specific reason for this. I usually try to follow the existing ways in the project. It has been implemented in this way in other places in the project. I think using both should work just fine.